docs/docs-content/clusters/public-cloud/aws/eks.md

```diff
@@ -1,35 +1,43 @@
 
 ## Prerequisites
 
-The following prerequisites must be met before deploying a cluster to AWS:
+- Access to an AWS cloud account.
 
-- Access to an AWS cloud account 
 - Palette integration with AWS account. Review [Add AWS Account](add-aws-accounts.md) for guidance.
-- An infrastructure cluster profile for AWS EKS. Review [Create an Infrastructure Profile](../../../profiles/cluster-profiles/create-cluster-profiles/create-infrastructure-profile.md) for guidance.
-- An [EC2 Key Pair](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html) in the target region.
-- Palette creates compute, network, and storage resources in AWS during the provisioning of Kubernetes clusters. Ensure there is sufficient capacity in the preferred AWS region for the creation of the following resources:
-    - vCPU
-    - VPC
+
+<!-- - A Virtual Private Cloud (VPC) with at least two subnets in different Availability Zones (AZs). Palette requires two AZs within the VPC that you specify when creating the EKS cluster. -->
+
+- An infrastructure cluster profile for AWS EKS. When you create the profile, ensure you choose **EKS** as the **Managed Kubernetes** cloud type. Review [Create an Infrastructure Profile](../../../profiles/cluster-profiles/create-cluster-profiles/create-infrastructure-profile.md) for guidance.
+
+- An EC2 key pair for the target region that provides a secure connection to your EC2 instances. To learn how to create a key pair, refer to the [Amazon EC2 key pairs](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html) resource.
+
+- kubelogin installed. This is a kubectl plugin for Kubernetes OpenID Connect (OIDC) authentication, also known as `kubectl oidc-login`.
+
+- Palette creates compute, network, and storage resources in AWS when it provisions Kubernetes clusters. Ensure there is sufficient capacity in the preferred AWS region to create the following resources:
+    - Virtual CPU (vCPU)
+    - Virtual Private Cloud (VPC)
     - Elastic IP
     - Internet Gateway
     - Elastic Load Balancers
-    - NAT Gateway
+    - Network Address Translation (NAT) Gateway
 
 
 :::info
 
-The following tags should be added to the virtual private network (VPC) public subnets to enable automatic subnet discovery for integration with AWS load balancer service. Replace the value `yourClusterName` with your cluster's name.
+To enable automatic subnet discovery for integration with AWS load balancer service, you need to add tags to the the Virtual Private Cloud (VPC) public subnets. Use the AWS Tag Editor and specify the region and resource type. Then, add the following tags. Replace the value `yourClusterName` with your cluster's name. Refer to [AWS Tag Editor](https://docs.aws.amazon.com/tag-editor/latest/userguide/tag-editor.html) for more information.

```

Comment:
"with **the** AWS load balancer service" 
---
"To enable automatic subnet discovery for integration with AWS load balancer service, you need to add tags to the the Virtual Private Cloud (VPC) public subnets."
what about:
"To enable the automated creation of AWS external load balancers, it is required to tag all public subnets in the Virtual Private Cloud (VPC) public subnets."

Reply:
I'm not sure about this suggestion. It sounds like the subnets need to be discovered and tags are the means to locate them. I'll check with @karl-cardenas-coding.